packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -59,6 +59,8 @@ const ignoredOperations = [
   "getBlindedBlock", // https://github.com/ChainSafe/lodestar/issues/5699
   "getNextWithdrawals", // https://github.com/ChainSafe/lodestar/issues/5696
   "getDebugForkChoice", // https://github.com/ChainSafe/lodestar/issues/5700
+  /* Must support ssz response body */
+  "getLightClientUpdatesByRange", // https://github.com/ChainSafe/lodestar/issues/6841
```

Comment:
Ran this manually, json spec tests are passing but for now need to exclude the whole route as ssz tests are failing